src/traces/carpet/axis_defaults.js

```diff
@@ -166,13 +168,16 @@ module.exports = function handleAxisDefaults(containerIn, containerOut, options)
     if(!showGrid) {
         delete containerOut.gridcolor;
         delete containerOut.gridWidth;
+        delete containerOut.gridDash;
```

Comment:
This line should possibly use lowercase `.griddash`. 
Concerning the camelCase `gridWidth` on the line above, I opened #6151.